### PR TITLE
CART-89 mercury: Update to mercury 2.0.0

### PR DESCRIPTION
**Pull request by frostedcmos**
- Update to release 2.0.0

Signed-off-by: Alexander Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NAME       := mercury
 SRC_EXT    := bz2
-DL_VERSION := 2.0.0rc3
+DL_VERSION := 2.0.0
```
Comment (brianjmurrell) [inline Makefile:3]: Is this necessary any more?

File: mercury.spec [hunk 2]
```diff
@@ -1,7 +1,7 @@
-%global dl_version 2.0.0rc3
+%global dl_version 2.0.0
```
Comment (brianjmurrell) [inline mercury.spec:1]: I think we can get rid of this now and use `%{version}` in it's place.

**Review by brianjmurrell — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (frostedcmos) [reply Makefile]: i kept it initially in case we have 2.0.1rc1 or some such later, but removed now

File: mercury.spec (hunk 2 above)
Comment (frostedcmos) [reply mercury.spec]: done

File: mercury.spec [hunk 3]
```diff
@@ -45,7 +43,7 @@ Mercury devel
 
 %prep
 
-%autosetup -n mercury-%dl_version
+%autosetup -n mercury-%version
```
Comment (brianjmurrell) [inline mercury.spec:46]: FWIW, I think this is the default.  Not reason enough to repush unless you want to, but:
```suggestion
%autosetup
```

**Review by brianjmurrell — APPROVED** [review]

**Review by jolivier23 — APPROVED** [review]

**Pull request merged**